### ISSUE TITLE
[Bug] Account Manager is buggy

### ISSUE THREAD

**Issue by DarkReaper231** [MeteorDevelopment/meteor-client#2042]
### Describe the bug

When you have thealtening alts or premium alts and you join any server and disconnect, you can't switch to another alt. You can switch to cracked ones but altening and premium alts get bugged. Not sure about Mojang accounts, only tested with Microsoft and Altening. 

### Steps to reproduce

```bash
Login to cracked alt, join server, disconnect and try to login to altening alt or microsoft account and it won't work
```

### Meteor Version

built from latest source using gradle

### Minecraft Version

1.18.1

### Operating System

Linux

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by Justgamer101** [comment]
dont build it and you haven't run it on the latest dev build. and why tf does everyone think building is safer? it's not like ur actually reading any code to check.

**Comment by SIMULATAN** [comment]
> dont build it and you haven't run it on the latest dev build. (unlike ur checkmark says) and why tf does everyone think building is safer? it's not like ur actually reading any code to check.

That's true but MANUAL BUILDS could contain malware / you could've swapped out the code. Unfortunately, those lil kids don't get what a "CI" is and thus don't understand that the full building process is open and transparent.

**Comment by DarkReaper231** [comment]
> > dont build it and you haven't run it on the latest dev build. (unlike ur checkmark says) and why tf does everyone think building is safer? it's not like ur actually reading any code to check.
> 
> That's true but MANUAL BUILDS could contain malware / you could've swapped out the code. Unfortunately, those lil kids don't get what a "CI" is and thus don't understand that the full building process is open and transparent.

Uh I am using built jars from source because I forked it and removed nested baritone in it so I can swap it with altoclef instead in mods folder duh https://github.com/DarkReaper231/meteor-client

And I don't see reason why would that matter if you use stable release or dev version. The bug must be resolved anyway.

**Comment by SIMULATAN** [comment]
> And I don't see reason why would that matter if you use stable release or dev version. The bug must be resolved anyway.
Because dev versions are newer than the stable releases and thus might include fixes that the stable version doesn't have?
